openpype/hosts/fusion/hooks/pre_fusion_profile_hook.py

```diff
@@ -0,0 +1,129 @@
+import os
+import shutil
+import platform
+from pathlib import Path
+from openpype.lib import PreLaunchHook
+from openpype.hosts.fusion import FUSION_HOST_DIR, get_fusion_profile_number
+
+
+class FusionCopyPrefsPrelaunch(PreLaunchHook):
+    """Prepares local Fusion profile directory, copies existing Fusion profile
+    """
+
+    app_groups = ["fusion"]
+    order = 2
+
+    def get_fusion_profile_name(self, app_version) -> str:
+        # Returns 'Default', unless FUSION16_PROFILE is set
+        return os.getenv(f"FUSION{app_version}_PROFILE", "Default")
+
+    def check_profile_variable(self, app_version) -> Path:
+        # Get FUSION_PROFILE_DIR variable
+        fusion_profile = self.get_fusion_profile_name(app_version)
+        fusion_var_prefs_dir = os.getenv(f"FUSION{app_version}_PROFILE_DIR")
+
+        # Check if FUSION_PROFILE_DIR exists
+        if fusion_var_prefs_dir and Path(fusion_var_prefs_dir).is_dir():
+            fu_prefs_dir = Path(fusion_var_prefs_dir, fusion_profile)
+            self.log.info(f"{fusion_var_prefs_dir} is set to {fu_prefs_dir}")
+            return fu_prefs_dir
+
+    def get_profile_source(self, app_version) -> Path:
+        """Get Fusion preferences profile location.
+        See Per-User_Preferences_and_Paths on VFXpedia for reference.
+        """
+        fusion_profile = self.get_fusion_profile_name(app_version)
+        profile_source = self.check_profile_variable(app_version)
+        if profile_source:
+            return profile_source
+        # otherwise get default location of the profile folder
+        fu_prefs_dir = f"Blackmagic Design/Fusion/Profiles/{fusion_profile}"
+        if platform.system() == "Windows":
+            profile_source = Path(os.getenv("AppData"), fu_prefs_dir)
+        elif platform.system() == "Darwin":
+            profile_source = Path(
+                "~/Library/Application Support/", fu_prefs_dir
+            ).expanduser()
+        elif platform.system() == "Linux":
+            profile_source = Path("~/.fusion", fu_prefs_dir).expanduser()
+        self.log.info(f"Got Fusion prefs file: {profile_source}")
```

Comment:
The folder might not exist.

```suggestion
        self.log.info(f"Locating source Fusion prefs directory: {profile_source}")
```